runtime/src/main/java/io/quarkiverse/logging/splunk/SplunkConfig.java

```diff
@@ -6,26 +6,32 @@
 
 import java.util.Map;
 
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithParentName;
 
 /**
  * Configuration for Splunk HEC logging
  */
-@ConfigRoot(phase = ConfigPhase.RUN_TIME, name = "log.handler.splunk")
-public class SplunkConfig {
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+@ConfigMapping(prefix = "quarkus.log.handler.splunk")
+public interface SplunkConfig {
```

Comment:
It's pity they changed to an interface, with concrete classes you could programmatically create your own config and write JMH benchmarks easily, now you need Quarkus to do so.
I'll check if there's quarkus JMH extension